pkg/operator/ceph/csi/template/cephfs/csi-cephfsplugin-provisioner-dep.yaml

```diff
@@ -119,6 +119,9 @@ spec:
             {{ if .CSIClusterName }}
             - "--clustername={{ .CSIClusterName }}"
             {{ end }}
+            {{ if .EnableCSIAddonsSideCar }}
```

Comment:
how this will be enabled, do we need a cluster.yaml settings

Reply:
After deploying csi-addon manifests, we do 
`kubectl` patch cm rook-ceph-operator-config -n<namespace> -p $'data:\n "CSI_ENABLE_CSIADDONS": "true"'
if crds found, then it enables it (https://github.com/rook/rook/pull/12487)

Reply:
So can we have this information documented

Reply:
And another question I should have asked in the other PR, who create the csi-addon crds?

Reply:
We have it documented here, https://github.com/rook/rook/blob/master/Documentation/Storage-Configuration/Ceph-CSI/ceph-csi-drivers.md?plain=1#L117 
I can add more detail about this feature on a different PR where Network fence will be coded? 
Also, we can enable if by default too WDYT?


>>And another question I should have asked in the other PR, who create the csi-addon crds?
Not sure, but user should be the one creating.